lib/utilities/requestUtils.js

```diff
@@ -0,0 +1,35 @@
+const { config } = require('../Config');
+const { ipCheck } = require('arsenal');
+
+/**
+ * getClientIp - Gets the client IP from the request
+ * @param {object} request - http request object
+ * @param {object} s3config - (optional) s3 config
+ * @return {string} - returns client IP from the request
+ */
+function getClientIp(request, s3config) {
+    const clientIp = (request.connection && request.connection.remoteAddress) ||
+        (request.socket && request.socket.remoteAddress) ||
```

Comment:
`request.socket` and `request.connection` seem to be the same thing, `request.socket` being the canonical one.

Reply:
@RachedBenMustapha yes, I checked the docs they are the same. So reverting to use only `request.socket`.